homeassistant/components/shelly/climate.py

```diff
@@ -547,6 +547,8 @@ class RpcBluTrvClimate(ShellyRpcEntity, ClimateEntity):
     _attr_hvac_modes = [HVACMode.OFF, HVACMode.HEAT]
     _attr_target_temperature_step = RPC_THERMOSTAT_SETTINGS["step"]
     _attr_temperature_unit = UnitOfTemperature.CELSIUS
+    _attr_has_entity_name = True
+    _attr_name = None
```

Comment:
The `_attr_name = None` needs to be in the constructor (`self._attr_name = None`)